opentelemetry/README.md

```diff
@@ -0,0 +1,63 @@
+# OpenTelemetry-Ktor Demo
+
+## Running
+To run a sample, first, execute the following command in an `opentelemetry` directory:
+```bash
+./gradlew :runWithDocker
```

Comment:
Mayne add here that Docker needs to be installed and running?

Reply:
Yes, good point, will add